### ISSUE TITLE
slow start show port used error

### ISSUE THREAD

**Issue by joeylin** [balderdashy/sails#2921]
My sails app start a little slow, it starts normally, but sails still shows port used error .￼ below is ERROR log
￼￼
debug: --------------------------------------------------------
debug: :: Mon May 11 2015 15:54:16 GMT+0800 (CST)

debug: Environment : docs-aliyun-inc
debug: Port        : 8001
debug: --------------------------------------------------------
call getFeedbackStatus

error: Server doesn't seem to be starting.
error: 
error: Troubleshooting tips:
error: 
error:  -> Is something else already running on port 8001 ?
error: 
error:  -> Are you deploying on a platform that requires an explicit hostname, like OpenShift?
error:     (Try setting the `explicitHost` config to the hostname where the server will be accessible.)
error:     (e.g. `mydomain.com` or `183.24.244.42`)

**Comment by tjwebb** [comment]
> error: -> Is something else already running on port 8001 ?

Well, is something else running on port 8001?

**Comment by sailsbot** [comment]
Thanks for posting, @joeylin.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
